crates/loam-cli/tests/it/build_clients/contracts.rs

```diff
@@ -165,11 +165,33 @@ soroban_token_contract.client = false
             .output()
             .expect("Failed to execute command");
 
-        println!("stderr: {}", String::from_utf8_lossy(&output3.stderr));
+        //println!("stderr: {}", String::from_utf8_lossy(&output3.stderr));
+        //println!("stdout: {}", String::from_utf8_lossy(&output3.stdout));
         // ensure contract hash change check works, should update in dev mode
         assert!(output3.status.success());
-        assert!(String::from_utf8_lossy(&output3.stderr)
-            .contains("🔄 Updating contract \"hello_world\""));
+        let message = String::from_utf8_lossy(&output3.stderr);
+        assert!(message.contains("🔄 Updating contract \"hello_world\""));
+        let Some(contract_id) = extract_contract_id(&message) else {
+            panic!("Could not find contract ID in stderr");
+        };
+        env.set_environments_toml(format!(
+            r#"
+production.accounts = [
+    {{ name = "alice" }},
+]
+
+[production.network]
+rpc-url = "http://localhost:8000/rpc"
+network-passphrase = "Standalone Network ; February 2017"
+
+[production.contracts]
+{}.client = true
+"#,
+            contract_id
```

Comment:
am I reading correctly that this renders the correctly-formatted environments.toml as:

```toml
[production.contracts]
CDL7EPDGP7EL52QTYU4RVP6LSMAVHGCNUGUAOI4YMBMIWXUTHUF3V3ZI.client = true
```

Reply:
add the id field like we discussed. I also added support for it in dev mode.